docs/framework/wcf/feature-details/creating-multicasting-applications-using-the-udp-transport.md

```diff
@@ -85,7 +85,7 @@ while (true)
  This code generates stock information and then uses the service contract IStockTicker to send multicast messages to call services listening on the correct UDP address.  
   
 ### UDP and Reliable Messaging  
- The UDP binding does not support reliable messaging because of the lightweight nature of the UDP protocol. If you need to confirm that messages are received by a remote endpoint, use a transport that supports reliable messaging like  HTTP or TCP. For more information about reliable messaging see https://go.microsoft.com/fwlink/?LinkId=231830  
+ The UDP binding does not support reliable messaging because of the lightweight nature of the UDP protocol. If you need to confirm that messages are received by a remote endpoint, use a transport that supports reliable messaging like  HTTP or TCP. For more information about reliable messaging see <https://go.microsoft.com/fwlink/?LinkId=231830>  
```

Comment:
nit

```suggestion
 The UDP binding does not support reliable messaging because of the lightweight nature of the UDP protocol. If you need to confirm that messages are received by a remote endpoint, use a transport that supports reliable messaging like  HTTP or TCP. For more information about reliable messaging, see <https://go.microsoft.com/fwlink/?LinkId=231830>.
```

Reply:
Manually applied your changes while making the other change from above. Thanks!